stable/fluentd/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: A Fluentd Elasticsearch Helm chart for Kubernetes.
 icon: https://raw.githubusercontent.com/fluent/fluentd-docs/master/public/logo/Fluentd_square.png
 name: fluentd
-version: 2.3.2
+version: 2.3.3
```

Comment:
I think that this is a new feature and not a bugfix, so `2.4.0`

Reply:
Ok, I've updated the version.